rubin_sim/maf/batches/scienceRadarBatch.py

```diff
@@ -732,10 +732,11 @@ def scienceRadarBatch(
         verbose=False,
     )
     plotDict = {"percentileClip": 95, "nTicks": 5}
+    # Run without DDF observations
     bundle = mb.MetricBundle(
         metric,
         snslicer,
-        "",
+        "note not like %DD%",
```

Comment:
Can you humor me and add the info_label as well, so that it says more simply "non-DD"  in the plot title/metric file, etc. instead of using this awkward sql constraint in those places?